### PR TITLE
Feature/29 edit instances

### PR DESCRIPTION
**Pull request by Munin33**
This pull request implements the editing of resource instances and fixes a bug when creatng resource types

### CODE REVIEW

File: src/router.ts [hunk 1]
```diff
@@ -42,6 +43,11 @@ export default new Router({
       name: 'resource',
       component: Resource,
     },
+    {
```
Comment (friedow) [inline src/router.ts]: Move this route above the 'resource' route to avoid route matching conflicts.

File: src/views/EditInstance.vue [hunk 2]
```diff
@@ -0,0 +1,90 @@
+<template>
+  <main>
+    <ViewHeader title="Choose new attribute values for the instance" :backLink="{ link: `/resources/${this.instanceId}` }"/>
```
Comment (friedow) [inline src/views/EditInstance.vue]: Please use a named route objects for navigation. (reference code inside `saveResourceInstance`).

File: src/views/EditInstance.vue [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    try {
+      await ResourceInstances.update(this.instanceId, this.updatedResourceInstance);
+      this.$notifications.create({
+        title: 'Resource has been updated.',
+        details: '',
+        level: NotificationLevel.Success,
+        timestamp: new Date(),
+      });
+      if (this.updatedResourceInstance.id) {
+        this.$router.push({ name: 'resource', params: { id: this.updatedResourceInstance.id } });
```
Comment (friedow) [inline src/views/EditInstance.vue:77]: 👍

**Review by friedow — CHANGES_REQUESTED** [review]

File: src/router.ts (hunk 1 above)
Comment (Munin33) [reply src/router.ts]: you are right, I changed the order of the routes.

File: src/views/EditInstance.vue (hunk 2 above)
Comment (Munin33) [reply src/views/EditInstance.vue]: Ah, I overlooked this one, thanks!

**Review by friedow — APPROVED** [review]

**Pull request merged**